code/controllers/subsystem/shuttles/supply.dm

```diff
@@ -287,7 +287,7 @@
 				temp += "<A href='?src=\ref[src];doorder=[supply_type]'>[N.name]</A> Cost: [N.cost]<BR>"		//the obj because it would get caught by the garbage
 	else if (href_list["doorder"])
 		if(world.time < reqtime)
-			say("[world.time - reqtime] seconds remaining until another requisition form may be printed.")
+			say("[round((reqtime - world.time)/10,0.1)] second\s remaining until another requisition form may be printed.")
```

Comment:
Well I assumed he was suggesting it was supposed to be the other way around, which would give you a negative number